minio/credentials/providers.py

```diff
@@ -418,9 +418,12 @@ def retrieve(self):
         url = self._custom_endpoint
         if self._token_file:
             if not url:
-                url = "https://sts.{0}{1}amazonaws.com".format(
-                    self._aws_region, "." if self._aws_region else "",
-                )
+                if self._aws_region:
+                    url = "https://sts.{0}.amazonaws.com".format(
+                        self._aws_region
+                    )
```

Comment:
Fit it into one line.

Reply:
It will be longer than 80 characters (pylint check) if it's one line.

Reply:
Uh, maybe, I can make it just 80 by introducing the default URL you commented about.

Reply:
Sorry, I'm not able to make it shorter than 80... Do you have any suggestions to avoid the pylint error?

Reply:
```diff
diff --git a/minio/credentials/providers.py b/minio/credentials/providers.py
index 28b17e3..ba7fd76 100644
--- a/minio/credentials/providers.py
+++ b/minio/credentials/providers.py
@@ -417,10 +417,9 @@ class IamAwsProvider(Provider):
 
         url = self._custom_endpoint
         if self._token_file:
-            if not url:
-                url = "https://sts.{0}{1}amazonaws.com".format(
-                    self._aws_region, "." if self._aws_region else "",
-                )
+            url = "https://sts.amazonaws.com"
+            if self._aws_region:
+                url = "https://sts.{0}.amazonaws.com".format(self._aws_region)
 
             provider = WebIdentityProvider(
                 lambda: _get_jwt_token(self._token_file),
```